changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.4.0 | 2019-12-05
+
+* Add calibration data as attribute of force channels.
+* Take into account slicing of channels when returning force calibration data.
```

Comment:
I don't think that this is new compared to v0.3. This is just part for the calibration feature stated above.

Reply:
Removed.